src/Pages/Landing/Landing.js

```diff
@@ -6,8 +6,8 @@ function Landing() {
     <div>
       <main className='main-content'>
         <Header />
-        <p>Landing page</p>
         <Login />
+        Landing
```

Comment:
Bare text is not recommended. Also the landing text could be completly removed. But rembember to change unit test also when removing it